packages/astro/src/runtime/client/dev-toolbar/apps/audit/a11y.ts

```diff
@@ -496,13 +496,17 @@ export const a11y: AuditRuleWithSelector[] = [
 			if (is_semantic_role_element(role, element.localName, getAttributeObject(element))) {
 				return;
 			}
-			const { requiredProps } = roles.get(role)!;
-			const required_role_props = Object.keys(requiredProps);
-			const missingProps = required_role_props.filter((prop) => !element.hasAttribute(prop));
-			if (missingProps.length > 0) {
-				(element as any).__astro_role = role;
-				(element as any).__astro_missing_attributes = missingProps;
-				return true;
+			
+			const elementRoles = role.split(/\s+/) as ARIARoleDefinitionKey[];
```

Comment:
nit: it would benefit the performance if we moved this regex to a top-level const, so we don't need to create a new regex every single time. 

```js
const WHITESPACE_REGEX = /s+/;
// ...

role.split(WHITESPACE_REGEX);
```

Reply:
Thank you for your review. I have made the necessary changes. Thank you very muchI have made the modifications as required, please help me review again, thank you.